src/Symfony/Component/Messenger/composer.json

```diff
@@ -22,7 +22,7 @@
     },
     "require-dev": {
         "psr/cache": "^1.0|^2.0|^3.0",
-        "symfony/console": "^5.4|^6.0",
+        "symfony/console": "^6.3",
```

Comment:
The `SignalableCommandInterface::handleSignal()` methods signature changed in 6.3, it's either this or a conditional trait I guess.